### DISCUSSION TITLE
测试了一下只能下载480P，请问该如何解决

### DISCUSSION THREAD

**Discussion by Bonespy2016** [Nemo2011/bilibili-api#239] [Q&A]
测试了一下，参照API文档，先用示例脚本登录，再运行了视频下载的示例脚本，可是只有480p分辨率。换了几个视频地址都这样。我看示例中有detect_best_streams可以自动选择最高分辨率，没有排查出错误，想问下大佬们如何获得480P以上分辨率的文件呢？

**Comment by z0z0r4** [comment]
给个返回值

**Reply by z0z0r4** [reply]
提供代码？
